src/core/jit/context.rs

```diff
@@ -5,42 +5,25 @@ use super::Request;
 use crate::core::ir::ResolverContextLike;
 
 /// Rust representation of the GraphQL context available in the DSL
-#[derive(Getters)]
+#[derive(Getters, Debug)]
 pub struct Context<'a, Input, Output> {
     request: &'a Request<Input>,
-    parent: Option<&'a Output>,
```

Comment:
We will need this field when we integrate support for bulk APIs without using a dataloader.

Reply:
For now we can keep it deleted.